packages/video_player/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayerPlugin.java

```diff
@@ -56,6 +60,16 @@ public static void registerWith(io.flutter.plugin.common.PluginRegistry.Registra
 
   @Override
   public void onAttachedToEngine(FlutterPluginBinding binding) {
+    if (android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+      try {
+        HttpsURLConnection.setDefaultSSLSocketFactory(new CustomSSLSocketFactory());
+      } catch (KeyManagementException e) {
+        e.printStackTrace();
+      } catch (NoSuchAlgorithmException e) {
+        e.printStackTrace();
```

Comment:
Is there any information we can add here to help developers out? Particularly if they're seeing this in crash reporting in analytics.